xterm/index.cgi

```diff
@@ -282,9 +282,7 @@ my $term_script = <<EOF;
 		
 		// On resize event triggered by fit()
 		term.onResize(function(e) {
-			// \033[8;(40);(100)t
-			//socket.send('\\\\033\\\\[8;('+e.rows+');('+e.cols+')t');
-			socket.send('___RESIZE___ '+e.rows+' '+e.cols);
+			socket.send('\\\\033[8;('+e.rows+');('+e.cols+')t');
```

Comment:
Why this escape sequence? Is it interpreted by the shell?

Reply:
No, we will interpret on shell server.

Reply:
It's somewhat expected sequence though (used in PR). `___RESIZE___` theoretically could be used for something else. Where as this one is expected to do resize only. I realize that we don't send it to the terminal, and we send it over the socket to the shell. Yet I find it safer, as anyone calling it would only expect to do a resize on the terminal, and nothing else.

Reply:
Ok, makes sense